### PR TITLE
Reconstruct pc sanity test plan (BugFix)

### PR DESCRIPTION
**Pull request by xanthein**
## Description
Create separate pc-sanity test plan for different LTS release

## Resolved issues
[OEX86-105](https://warthogs.atlassian.net/browse/OEX86-105)

## Tests

[OEX86-105]: https://warthogs.atlassian.net/browse/OEX86-105?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by xanthein** [comment]
@KaiChuan-Hsieh @binli can you help to review the PR

File: contrib/pc-sanity/units/pc-sanity/pc-sanity.pxu [hunk 1]
```diff
@@ -1,8 +1,8 @@
-id: pc-sanity-smoke-test
-_name: pc-sanity-smoke-test (Ubuntu Desktop)
+id: pc-sanity-base
+_name: pc-sanity-base
 unit: test plan
 _description:
-    This test plan is for smoke test as sanity before release image.
+    pc-sanity tests
```
Comment (KaiChuan-Hsieh) [inline contrib/pc-sanity/units/pc-sanity/pc-sanity.pxu:5]: Can you add more description here? for example:

1. what are in the plan, who contribute it.
2. what is the plan for, who should include it or in what circumstance we should include it.

File: contrib/pc-sanity/units/pc-sanity/pc-sanity.pxu [hunk 2]
```diff
@@ -81,7 +81,27 @@ exclude:
...
     com.canonical.certification::pc-sanity-base
-    com.canonical.certification::client-cert-auto
+    com.canonical.certification::client-cert-desktop-22-04-automated
+    com.canonical.certification::power-management-cpu-cstate-cert-automated
+bootstrap_include:
+    device
+    graphics_card
+    cpuinfo
+
+id: pc-sanity-smoke-test-24-04
```
Comment (KaiChuan-Hsieh) [inline contrib/pc-sanity/units/pc-sanity/pc-sanity.pxu:91]: I think we can remove no gpu switching plan and make no dgpu switching become default. What do you think?

**Review by KaiChuan-Hsieh — COMMENTED** [review]
Please see my comment in-line, thanks.

File: contrib/pc-sanity/units/pc-sanity/pc-sanity.pxu (hunk 2 above)
Comment (xanthein) [reply contrib/pc-sanity/units/pc-sanity/pc-sanity.pxu]: yes, I think we can make no dgpu switching become default

**Pull request merged**